src/gorilla.c

```diff
@@ -285,38 +300,40 @@ static ChunkResult appendInteger(CompressedChunk *chunk, timestamp_t timestamp)
        * The second value is a compressed representation of the value with the `length`
          encoded by the first value. Compression is done using `int2bin`.
      */
-    binary_t *bins = chunk->data;
-    globalbit_t *bit = &chunk->idx;
+ 
+    binary_t *bins = chunk->data_ts;
+    globalbit_t *bit = &chunk->idx_ts;
     if (doubleDelta.i == 0) {
-        CHECKSPACE(chunk, 1 + 1); // CHECKSPACE adds 1 as minimum for double space
+        CHECKSPACE_TS(chunk, 1); 
```

Comment:
@slice4e  This change is ok, the + 1 is optimization for checking space for the also.